src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -79,13 +79,22 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
     return index;
   }, [activeIndex, labels.length]);
 
+  const tablistRef = React.useRef<HTMLUListElement>(null);
+
   const [currentIndex, setCurrentIndex] = React.useState(getValidIndex());
+  const [stripeStyle, setStripeStyle] = React.useState<React.CSSProperties>({});
 
   React.useEffect(() => {
```

Comment:
We might need to use `useLayoutEffect` to avoid random bugs when stripe is not positioned correctly or only after some time.

Reply:
Changed